server/routes/groups.js

```diff
@@ -0,0 +1,63 @@
+import uuid from 'uuid';
+import { Router } from 'express';
+
+import { matchWithApps, saveGroup, deleteGroup } from '../lib/groups';
+
+export default (auth0, storage) => {
+  const api = Router();
+
+  api.get('/', (req, res, next) => {
```

Comment:
Should I use the same `read:applications` scope or a different one like `read:groups`?